core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -45,6 +45,34 @@ oppia.factory('explorationData', [
     // Put exploration variables here.
     var explorationData = {
       explorationId: explorationId,
+      autosaveChangeList: function(changeList, successCallback, errorCallback) {
+        var draftAutosaveUrl = '/createhandler/autosave_draft/' + explorationId;
```

Comment:
No I just meant you can write 
'/createhandler/autosave_draft/' + explorationId
directly where you wrote
draftAutosaveUrl

It's generally not worth defining variables for simple things that only get used once.